src/components/actividades/AltaModificacionActividad.js

```diff
@@ -40,28 +40,30 @@ export default function AltaActividad(props) {
   const history = useHistory();
   const { create, update } = useApi('actividades');
 
-  ValidatorForm.addValidationRule(
-    'fechaInicioValida',
-    (value) => DateTime.fromISO(value) > DateTime.local()
-  );
-
-  ValidatorForm.addValidationRule(
-    'fechaFinValida',
-    (value) => value > actividad.fechaHoraInicio
-  );
-
+  const espacios = useRecoilValue(todosLosEspacios);
+  const carreras = useRecoilValue(todasLasCarreras);
   const [actividad, setActividad] = useState(actividadDB.data);
   const {
     espacioId,
     nombre,
     fechaHoraInicio,
     fechaHoraFin,
     responsable,
-    dniResponsable,
-    activa,
+    telefonoDeContactoResponsable,
     restriccionId,
   } = actividad;
 
+  const carreraSeleccionada = find(propEq('id', restriccionId))(carreras);
```

Comment:
Me gusta que le vayan agarrando gustito a Ramda. La lista también se puede pasar como segundo parámetro, tal vez más limpio porque usás menos paréntesis:


```suggestion
  const carreraSeleccionada = find(propEq('id', restriccionId), carreras);
```